domFoundations/events,storage,attributes/initialPageRenderer.js

```diff
@@ -0,0 +1,21 @@
+import { loadData } from "./storage.js";
+
+export const backgr = document.querySelector('body');
+export const dateField = document.querySelector('.dateOutput');
+export const btnOnOff = document.querySelector('.btnOnOff');
+export const dateOnOff = document.querySelector('.dateOnOff');
+export const dateText = document.querySelector('.date');
+export const timeText = document.querySelector('.time');
+export const textOutputdocument= document.querySelector('.dateOutput');
+
+export const startStateRenderer = () => {
+    backgr.style.backgroundColor = 'background-color' in localStorage ? loadData('background-color') : '#FFF';
+    dateField.style.color = 'date-color' in localStorage ? loadData('date-color') : '#000';
```

Comment:
the logic of checking property in local storage is better to hide inside the `loadData`. And you can have an interface like this (with default value):
```
loadData('background-color', '#FFF')
```

Reply:
Done